extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,93 @@
     opacity: .4;
   }
 }
+
+/* Disclaimer CSS rules */
+.i-amphtml-story-interactive-disclaimer {
+  --i-amphtml-story-interactive-disclaimer-animation-duration: 0.15s !important;
+  position: absolute !important;
+  color: black !important;
+  bottom: calc(100% - 1.5em) !important;
+  right: 0 !important;
+  z-index: 1 !important;
+}
+
+.i-amphtml-story-interactive-container:not(.i-amphtml-story-interactive-active) .i-amphtml-story-interactive-disclaimer {
+  display: none !important;  /* Disable the disclaimer if not on active page to prevent tabbing */
+}
+
+.i-amphtml-story-interactive-disclaimer-alert {
+  width: 1.75em !important;
+  height: 1.75em !important;
```

Comment:
buttons have a system font by default that breaks relative em sizing.
`font-size: inherit;` can fix this.
<img width="444" alt="Screen Shot 2020-12-15 at 2 30 29 PM" src="https://user-images.githubusercontent.com/3860311/102263296-1ae9e800-3ee2-11eb-89eb-11f12cf37439.png">